yt_dlp/extractor/tvw.py

```diff
@@ -1,7 +1,8 @@
 import json
+import urllib
 
 from .common import InfoExtractor
-from ..utils import clean_html, remove_end, unified_timestamp, url_or_none
+from ..utils import clean_html, remove_end, unescapeHTML, unified_timestamp, url_or_none
 from ..utils.traversal import traverse_obj
```

Comment:
For my changes from below to work
```suggestion
from ..utils import (
    clean_html,
    extract_attributes,
    parse_qs,
    remove_end,
    unified_timestamp,
    url_or_none,
)
from ..utils.traversal import find_element, require, traverse_obj
```